python/pyarrow/tests/test_schema.py

```diff
@@ -681,7 +681,8 @@ def test_schema_sizeof():
         pa.field('bar', pa.string()),
     ])
 
-    assert sys.getsizeof(schema) > 30
+    # Note: pa.schema is twice as large on 64-bit systems
+    assert sys.getsizeof(schema) > 30 if sys.maxsize > 2**32 else 15
```

Comment:
Hmm, let's avoid any ambiguities due to operator precedence here:
```suggestion
    assert sys.getsizeof(schema) > (30 if sys.maxsize > 2**32 else 15)
```

Reply:
Done, also for the other one.